src/lib/certmap/sss_certmap_ldap_mapping.c

```diff
@@ -228,14 +228,19 @@ int check_digest_conversion(const char *inp, const char **digest_list,
     bool colon = false;
     bool reverse = false;
     char *c;
+    size_t len;
 
     sep = strchr(inp, '_');
 
     for (d = 0; digest_list[d] != NULL; d++) {
         if (sep == NULL) {
             cmp = strcasecmp(digest_list[d], inp);
         } else {
-            cmp = strncasecmp(digest_list[d], inp, (sep - inp -1));
+            len = sep - inp;
```

Comment:
Maybe `len` could be computed only once outside of the loop.

Reply:
Hi,

thanks, good idea, fixed in latest version.

bye,
Sumit